README.md

```diff
@@ -147,7 +147,7 @@ script.
 
 Build the project (Optional: add `-DPERFTOOLS_PROFILER=True/False` and `-DALLOW_SHUTDOWN=True/False`)
 
-    cmake -DCMAKE_BUILD_TYPE=Release .. && make -j
+    cmake -DCMAKE_BUILD_TYPE=Release .. && make -j $(nproc)
```

Comment:
A rule of thumb for a good approximation of `-j` is usually `<NUM CORES> + 1`, some also use `<NUM CORES> * range(1.5, 2.0)`.
For `<NUM CORES> + 1` and `<NUM CORES> * 1.5` the main idea is the following:
You'll have `<NUM CORES>` for the compilation and and additional thread for management of I/O activity. The latter thread is often in idle mode, so that thread will lead to now problem.
If the hardware supports hyperthreading one can also use `<NUM CORES> * 2.0`. 

But, as usual: Such things highly depend on the underlying hardware.